interface/wx/choice.h

```diff
@@ -15,6 +15,8 @@
     @beginStyleTable
     @style{wxCB_SORT}
            Sorts the entries alphabetically.
+    @style{wxCH_BUTTONS}
```

Comment:
Shouldn't this be a style of `wxToolBar`? `wxChoice` really isn't supposed to look like this IMO.